### ISSUE TITLE
DigiTrust UserId module needs to check gdprConsent

### ISSUE THREAD

**Issue by goosemanjack** [prebid/Prebid.js#4674]
## Type of issue
This is a bug where behavior of the Prebid integration differs from the baseline DigiTrust library.

## Description
The DigiTrust integration should be specifically checking the transparency and consent framework for the DigiTrust publisher Id. This is in addition to the normal GDPR checks that Prebid performs.

## Other information
View this page for more information.
https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md

**Comment by goosemanjack** [comment]
A new PR has been created to address this ticket.
PR #4698 
https://github.com/prebid/Prebid.js/pull/4698

**Comment by goosemanjack** [comment]
Touching to un-stale. PR is waiting on review.

**Comment by patmmccann** [comment]
This is related to #4747

**Comment by goosemanjack** [comment]
This issue has been resolved and merged.
